### ISSUE TITLE
Anonymize scripts fails if WD is not the same as the location of the .ppj file

### ISSUE THREAD

**Issue by MrOctopus** [fireundubh/pyro#9]
Tested with the sse.ppj file in [repo](https://github.com/MrOctopus/nl_mcm/tree/main/main):

Running Pyro from the top directory will compile and zip the output:
```pyro --access-token TOKEN .\main\sse.ppj```
However, it will fail to anonymize the scripts and print the following warning "Cannot locate file to anonymize:"

Moving the WD to main and running Pyro again will compile, anonymize and zip the output successfully:
```pyro --access-token TOKEN .\sse.ppj```

**Comment by fireundubh** [comment]
Cannot reproduce.

```
F:\.virtualenvs\pyro\Scripts\python.exe F:/projects/pyro/pyro/__main__.py ".\Auto Loot\AutoLoot.pyroproject" --no-implicit-imports --no-incremental-build
2021-04-06 17:50:21,776 [WARN] Using working directory: "F:\repos\mods\fallout4"
2021-04-06 17:50:21,776 [WARN] Using input path: "F:\repos\mods\fallout4\.\Auto Loot\AutoLoot.pyroproject"
...
2021-04-06 17:50:27,421 [INFO] Anonymized "F:\repos\mods\fallout4\Auto Loot\scripts\AutoLoot\Fragments\Terminals\TERM_AutoLoot_100_04000FFC.pex"...
2021-04-06 17:50:27,422 [INFO] Anonymized "F:\repos\mods\fallout4\Auto Loot\scripts\AutoLoot\Fragments\Terminals\TERM_AutoLoot_210_All_04000FFE.pex"...
...
2021-04-06 17:50:27,804 [INFO] Compilation time: 5.602s (0.106s/script) - 53 succeeded, 0 failed (53 scripts)
2021-04-06 17:50:27,804 [INFO] DONE!
```

**Comment by MrOctopus** [comment]
It seems that yours is successful because your Output is not relative.

AutoLoot:
```
Output="@modpath\scripts"
<Variable Name="modpath" Value="F:\repos\mods\fallout4\@modname"/>
```

nl_mcm:
```
Output="@MCM\scripts"
<Variable Name="MCM" Value="."/>
```

**Comment by MrOctopus** [comment]
Fixed in 1414acb.
